frontend/src/component/Activity/ActivityList.jsx

```diff
@@ -115,6 +115,9 @@ const ActivityList = (props) => {
     return <div className='activityList__item' data-cy='activityList__item' key={activity.id}>{component}</div>
   }
 
+  // MB - 2021-05-26 - this code is duplicated for activityDisplayFilter, in withActivity
```

Comment:
```suggestion
  // FIXME - MB - 2021-05-26 - this code is duplicated for activityDisplayFilter, in withActivity
```

Change to be made at frontend/src/container/withActivity.jsx too.